website/src/pages/index.tsx

```diff
@@ -223,13 +223,19 @@ function Extensibility() {
             <FontAwesomeIcon size="3x" icon={faPlug} className="ml-2 mb-4 text-gray-800 dark:text-gray-200" />
           </div>
 
-          <p className="leading-relaxed">Container engines are plugged through extension points.</p>
-          <p className="leading-relaxed">JavaScript extensions can contribute new behaviour</p>
-          <p className="leading-relaxed">Reuse existing extensions directly in Podman Desktop</p>
+          <p className="leading-relaxed text-base">
+            Container engines are plugged through extension points.
+            <br />
+            JavaScript extensions can contribute new behaviour
+            <br />
+            Reuse existing extensions directly in Podman Desktop
+          </p>
```

Comment:
Here it looks like more a list, so you could use `<ul><li>` (and by default there is no bullet, it's up to you to choose)
https://tailwindcss.com/docs/list-style-type